styles/typographie.md

```diff
@@ -78,6 +78,15 @@ title: Typographie - Style de Canada.ca
    Corps du texte : Noto sans, 20px, texte régulier
   </li>
  </ul>
+   <h2>Titre principal de la page</h2>
+  <p>Lorsque l’en-tête H1 est le titre principal d’une page, il est souligné d’une barre rouge conformément à l’image de marque de Canada.ca.</p>
+  <p>Spécifications de la barre rouge (gc-thickline)&nbsp;:</p>
```

Comment:
Please change to:

Spécifications de la barre rouge (auparavant connue sous le nom de gc-thickline) :

Reply:
done